nixos/modules/services/networking/syncplay.nix

```diff
@@ -71,10 +83,20 @@ in
       after       = [ "network-online.target" ];
 
       serviceConfig = {
-        ExecStart = "${pkgs.syncplay}/bin/syncplay-server ${escapeShellArgs cmdArgs}";
         User = cfg.user;
         Group = cfg.group;
       };
+
+      script = ''
+        cmdArgs=(${escapeShellArgs cmdArgs})
+        ${lib.optionalString (cfg.passwordFile != null) ''
+          cmdArgs+=(
+            "--password"
+            "$(cat ${lib.escapeShellArg cfg.passwordFile})"
```

Comment:
Possibly it would be a better idea to set the password through the `SYNCPLAY_PASSWORD` environment variable to reduce the amount of places the password is leaked.

Reply:
Oh damn, I didn't realize that! I even opened [an issue](https://github.com/Syncplay/syncplay/issues/519) in the syncplay issue tracker saying that there's no way to specify the password without leaking it to other users!

Reply:
Now making use of `SYNCPLAY_PASSWORD` and also systemd's `LoadCredentials`, which is nice because then the `passwordFile` doesn't need to be accessible by the syncplay user.